src/Microsoft.Azure.ServiceBus/ServiceBusConnectionStringBuilder.cs

```diff
@@ -20,8 +20,10 @@ public class ServiceBusConnectionStringBuilder
         static readonly string SharedAccessKeyNameConfigName = "SharedAccessKeyName";
         static readonly string SharedAccessKeyConfigName = "SharedAccessKey";
         static readonly string EntityPathConfigName = "EntityPath";
+        static readonly string TransportTypeConfigName = "TransportType";
```

Comment:
Should these be `const`s instead? (I know this specific is added to what was there already).